documentatie/use-cases/GSB-subfuncties.md

```diff
@@ -0,0 +1,62 @@
+# Use cases GSB - subfunctie,  vis 🐟
+
+## De invoerder handelt de fout(en) af
+
+__niveau:__ subfunctie, vis, 🐟
+
+### Hoofdscenario en uitbreidingen
+
+__trigger:__ De controles geven een foutmelding vanwege de [validatieregels voor fouten](./GSB-validatieregels-plausibiliteitschecks.md#validatieregels-geven-fouten).
+
+*Foutmelding*: De ingevoerde waardes kunnen niet correct zijn. Bijvoorbeeld: het totaal van de stemmen op een lijst komt niet overeen met de som van de stemmen van de kandidaten op die lijst.
+
+__hoofdscenario__:  
+Voor elke foutmelding:  
+
+1. De invoerder controleert de foutmelding.
+2. De invoerder constateert dat hij/zij een fout heeft gemaakt in de invoer.
+3. De invoerder corrigeert de fout in de invoer.
+
+__uitbreidingen__:  
+2a. De invoerder stelt een fout op het PV vast en kan de foutmelding niet oplossen:  
+&emsp; 2a1. De invoerder meldt de fout op het PV bij de coördinator.  
+&emsp; 2a2. De coördinator stuurt het PV terug in het proces.  
+&emsp; 2a3. De invoerder breekt de invoer af.  
+&emsp; 2a4. De applicatie verwijdert de ingevoerde data.  
+
+### Open punten
+
+- Als de coördinator het PV terugstuurt in het proces, naar welk punt dan precies?
+
+## De invoerder handelt de waarschuwing(en) af
+
+__niveau:__ subfunctie, vis, 🐟
+
+### Hoofdscenario en uitbreidingen
+
+__trigger:__ De controles geven een waarschuwing vanwege de [plausibiliteitschecks](./GSB-validatieregels-plausibiliteitschecks.md#plausibiliteitschecks-geven-waarschuwingen).
+
+*Waarschuwing*: De ingevoerde waardes zijn mogelijk niet correct. Bijvoorbeeld: er is een groot aantal blanco stemmen of de tweede invoer klopt niet met de eerste invoer.
+
+__hoofdscenario__:  
+Voor elke waarschuwing:  
+
+1. De invoerder controleert de waarschuwing.
+2. De invoerder constateert dat de invoer klopt met het PV.
+3. De invoerder meldt de waarschuwing bij de coördinator.
```

Comment:
Dit is niet waar de teksten van de waarschuwingen nu op sturen. Die zijn: controleer het papier, als je invoer klopt met het papier, ga dan verder. Stap 3 en 4 moeten hier wat mij betreft weg. 

Wat we - voor de coördinator - wel willen gaan maken, is dat waarschuwingen voor hen zichtbaar worden op hun dashboard, zodat zij - nadat de eerste invoerder klaar is - de invoer kunnen bekijken en vergelijken met het PV om te beslisses of er aanvullend onderzoek gedaan moet worden.